pkg/cloud/services/iamauth/reconcile.go

```diff
@@ -61,6 +61,19 @@ func (s *Service) ReconcileIAMAuthenticator(ctx context.Context) error {
 		return fmt.Errorf("mapping iam node role: %w", err)
 	}
 
+	controllersRoleARN := fmt.Sprintf("arn:aws:iam::%s:role/controllers%s", accountID, iamv1.DefaultNameSuffix)
```

Comment:
Looking at this has made me wonder how handle the option of bootstrapping with a [prefix](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/d41f8cc91c992585fa4e1e0d693d408c89dbfb1d/cmd/clusterawsadm/cloudformation/bootstrap/template.go#L71) on the role names. But thats out of scope of this change.